packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -134,21 +135,84 @@ test.serial('writes status updates to vstorage', async t => {
   const wallet = await wd.provideSmartWallet(
     'agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78',
   );
-  const submitMockEvidence = (mockEvidence: CctpTxEvidence) =>
-    wallet.sendOffer({
-      id: 'submit-mock-evidence',
-      invitationSpec: {
-        source: 'agoricContract',
-        instancePath: ['fastUsdc'],
-        callPipe: [['makeTestPushInvitation', [mockEvidence]]],
-      },
-      proposal: {},
-    });
-  const mockEvidence1 = MockCctpTxEvidences.AGORIC_PLUS_OSMO();
-  const mockEvidence2 = MockCctpTxEvidences.AGORIC_PLUS_DYDX();
+  await wallet.sendOffer({
+    id: `submit-mock-evidence-dydx`,
+    invitationSpec: {
+      source: 'agoricContract',
+      instancePath: ['fastUsdc'],
+      callPipe: [
+        ['makeTestPushInvitation', [MockCctpTxEvidences.AGORIC_PLUS_DYDX()]],
```

Comment:
I'd like to get rid of `makeTestPushInvitation` soon. Is it important to use it here? What are your thoughts on when to stop using it?

I see you're using the real continuing invitation API below.

Reply:
Removed this test altogether since we check vstorage in the advancement test too now